packs/data/feats.db/diverse-lore.json

```diff
@@ -19,7 +19,24 @@
         "prerequisites": {
             "value": []
         },
-        "rules": [],
+        "rules": [
+            {
+                "domain": "skill-ckeck",
+                "key": "RollOption",
+                "label": "Diverse Lore",
+                "option": "diverse-lore-active",
+                "toggleable": true
+            },
+            {
+                "key": "FlatModifier",
+                "label": "Diverse Lore",
```

Comment:
```suggestion
```

Reply:
Just for my understanding, aren't these labels necessary to have a readable text in the roll options dialog?

Reply:
If a label is omitted, the name of the rule element's parent item is used instead.